src/packed/vector.rs

```diff
@@ -595,7 +595,7 @@ mod x86_64_avx2 {
     }
 }
 
-#[cfg(target_arch = "aarch64")]
+#[cfg(all(target_arch = "aarch64", target_feature = "neon", target_endian = "little"))]
```

Comment:
Are you sure that `target_feature = "neon"` is required here?

AIUI, `neon` is part of `aarch64`, so it should make sense to add this. (And perhaps I should add `target_feature = "sse2"` for `x86-64`.)

Reply:
I must admit that I'm a little uncertain about that, but it is my impression that it should have been sufficient to test for the presence of the "neon" feature (but wasn't), and that that feature is in fact not available when you run the CPU in BE mode.  However, given the testing I did appears to reveal that the "neon" feature is flagged as available also in BE mode, so if the `target_feature = "neon"` test was removed that would not change the ability to build it.